src/Storage.js

```diff
@@ -93,6 +93,24 @@ class Storage {
     const response = await axios.get(uri, { headers });
     return response.data;
   }
+
+  async listDataBySensor(deviceId, sensorId, query) {
+    validateId(deviceId);
```

Comment:
If we are validating the id, why shouldn't we validate sensor_id too?

Reply:
+1

Reply:
Done.